service/src/test/resources/requests/bibl-870970-25912233.json

```diff
@@ -0,0 +1,15 @@
+{
+    "agencyId": 870970,
+    "classifier": "basis",
+    "bibliographicRecordId": "25912233",
+    "repositoryId": "870970-basis:25912233",
+    "work": "work:1",
+    "unit": "unit:1",
+    "producerVersion": "version",
+    "deleted": false,
+    "indexKeys": {
+    },
+    "superceds": null,
```

Comment:
I don't know this format in detail, but could this be "supersedes"? Or would that require a bunch of code changes somewhere else?